### PR TITLE
translate page https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/ into Chinese

### PR DESCRIPTION
**Pull request by sarazqy**
translate page https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/ into Chinese

https://github.com/kubernetes/website/issues/34193

### CODE REVIEW

**Comment by sarazqy** [comment]
This is my new commit. Thanks for reviewing it when you have time @tengqm

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 1]
```diff
@@ -0,0 +1,1185 @@
+---
+api_metadata:
+  apiVersion: "batch/v1"
+  import: "k8s.io/api/batch/v1"
+  kind: "CronJob"
+content_type: "api_reference"
+description: "CronJob 代表单个 cron 作业的配置。"
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:7]:
```suggestion
description: "CronJob 代表单个定时作业（Cron Job）的配置。"
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 2]
```diff
@@ -0,0 +1,1185 @@
...
+title: "CronJob"
+weight: 10
+auto_generated: true
+---
+
+<!--
+---
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:14]: 删除14行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 3]
```diff
@@ -0,0 +1,1185 @@
...
+api_metadata:
+apiVersion: "batch/v1"
+import: "k8s.io/api/batch/v1"
+kind: "CronJob"
+content_type: "api_reference"
+description: "CronJob represents the configuration of a single cron job."
+title: "CronJob"
+weight: 10
+auto_generated: true
+---
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:24]: 删除24行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 4]
```diff
@@ -0,0 +1,1185 @@
...
+api_metadata:
+  apiVersion: "batch/v1"
+  import: "k8s.io/api/batch/v1"
+  kind: "CronJob"
+content_type: "api_reference"
+description: "CronJob 代表单个 cron 作业的配置。"
+title: "CronJob"
+weight: 10
+auto_generated: true
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:10]: 删除10行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 5]
```diff
@@ -0,0 +1,1185 @@
...
+`import "k8s.io/api/batch/v1"`
+
+
+## CronJob {#CronJob}
+
+<!--
+CronJob represents the configuration of a single cron job.
+-->
+
+CronJob 代表单个 cron 作业的配置。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:49]:
```suggestion
CronJob 代表单个定时作业（Cron Job）的配置。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 6]
```diff
@@ -0,0 +1,1185 @@
...
+- **kind**: CronJob
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->
+
+  标准对象的元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:65]: 注意缩进、留白、空行细节。下同。

```suggestion
  <!--
  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
  -->
  标准的对象元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 7]
```diff
@@ -0,0 +1,1185 @@
...
+
+- **spec** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobSpec" >}}">CronJobSpec</a>)
+
+<!--
+  Specification of the desired behavior of a cron job, including the schedule. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+
+  cron 作业所需的行为规范，包括计划。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:73]:
```suggestion
  定时作业的预期行为的规约，包括排期表（Schedule）。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 8]
```diff
@@ -0,0 +1,1185 @@
...
+
+- **status** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobStatus" >}}">CronJobStatus</a>)
+
+<!--
+  Current status of a cron job. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+  cron 作业的当前状态。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:80]:
```suggestion
  定时作业的当前状态。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 9]
```diff
@@ -0,0 +1,1185 @@
...
+
+## CronJobSpec {#CronJobSpec}
+
+<!--
+CronJobSpec describes how the job execution will look like and when it will actually run.
+-->
+
+CronJobSpec 描述了作业的执行方式和实际运行的时间。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:88]:
```suggestion
CronJobSpec 描述了作业的执行方式和实际将运行的时间。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 10]
```diff
@@ -0,0 +1,1185 @@
...
+
+<hr>
+
+<!--
+- **jobTemplate** (JobTemplateSpec), required
+
+  Specifies the job that will be created when executing a CronJob.
+-->
+
+- **jobTemplate** (JobTemplateSpec), required
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:98]: required 要翻译。下同。

```suggestion
- **jobTemplate** (JobTemplateSpec)，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 11]
```diff
@@ -0,0 +1,1185 @@
...
+
+  The time zone for the given schedule, see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones. If not specified, this will rely on the time zone of the kube-controller-manager process. ALPHA: This field is in alpha and must be enabled via the `CronJobTimeZone` feature gate.
+
+- **concurrencyPolicy** (string)
+
+  Specifies how to treat concurrent executions of a Job. Valid values are: - "Allow" (default): allows CronJobs to run concurrently; - "Forbid": forbids concurrent runs, skipping next run if previous run hasn't finished yet; - "Replace": cancels currently running job and replaces it with a new one
+-->
+
+  <a name="JobTemplateSpec"></a>
+  *JobTemplateSpec 描述了从模板创建作业时应具有的数据*
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:128]: 上面这种段落划分太大，很难评审，请重新组织后再评审。

单引号统一换成双引号，从斜体改为加粗。

```suggestion
  **JobTemplateSpec 描述了从模板创建作业时应具有的数据**
```

**Review by Sea-n — COMMENTED** [review]
The Netlify build is failed. Please take a look at [deploy log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62a85859be1b280007628e3b#L78-L79).

/hold

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md [hunk 12]
```diff
@@ -0,0 +1,1185 @@
...
+
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+    从此模板创建的作业的标准对象元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+  - **jobTemplate.spec** (<a href="{{< ref "../workload-resources/job-v1#JobSpec" >}}">JobSpec</a>)
```
[the remark this answers is not in the document]
Comment (sarazqy) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:134]: ok

[the remark this answers is not in the document]
Comment (sarazqy) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md:134]: I opened an issue, thanks for checking it
https://github.com/kubernetes/website/issues/34294

**Review by sarazqy — COMMENTED** [review]
I opened an issue, thanks for checking it
https://github.com/kubernetes/website/issues/34294

**Comment by sarazqy** [comment]
引用的页面没翻译，先翻译引用页

**Pull request closed without merging**